python/artm/__init__.py

```diff
@@ -0,0 +1,18 @@
+from artm_model import (
```

Comment:
Could you briefly explain why this change is useful? I'm really new to python so I simply don't know what to put in __init__.py... If there a good answer at stackoverflow or somewhere? I failed to find one...